README.md

```diff
@@ -232,51 +278,18 @@ Radsan contains a submodule with a fork of the `llvm-project`. This fork contain
       }
       ```
 
-# Building from source
-
-## Clang with RealtimeSanitizer
-
-Building RADSan-enabled clang is performed entirely within the fork of the
-llvm-project submodule. The llvm-project is a CMake project, and takes a bit of
-time to build. To minimise this build time, we recommend using the `ninja`
-build system, and configuring with the following CMake settings:
-
-```sh
-cd llvm-project
-mkdir build && cd build
-cmake -G Ninja \
-   -DCMAKE_BUILD_TYPE=Release \
-   -DBUILD_SHARED_LIBS=ON \
-   -DLLVM_ENABLE_PROJECTS="clang;compiler-rt" \
-   -DLLVM_TARGETS_TO_BUILD=Native \
-   ../llvm
-ninja -j8 clang compiler-rt llvm-symbolizer
-```
-
-If built successfully, `clang` should have appeared inside the `bin/` folder,
-and the radsan dynamic library should be in `lib/`:
+# Development
```

Comment:
Moved these under the "Development" heading, as it may not be of use to your average Joe